itests/eth_transactions_test.go

```diff
@@ -270,6 +271,76 @@ func TestContractInvocation(t *testing.T) {
 	require.EqualValues(t, ethtypes.EthUint64(0x1), receipt.Status)
 }
 
+func TestGetBlockByNumber(t *testing.T) {
+	blockTime := 100 * time.Millisecond
+	client, _, ens := kit.EnsembleMinimal(t, kit.MockProofs(), kit.ThroughRPC())
+
+	bms := ens.InterconnectAll().BeginMining(blockTime)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	// install contract
```

Comment:
Why are we deploying a contract / creating new EthAccounts here?

Reply:
Yeah, that was overkill.